src/main/resources/templates/layout/header.html

```diff
@@ -54,24 +54,24 @@
 <div class="container-fluid">
 
     <div id="errorPanel" class="alert alert-danger alert-dismissible fade show" style="display:none;" role="alert">
-        <strong>Error:</strong> <span id="errorText"></span>
-        <button type="button" class="close" aria-label="Close" onclick="$('.alert').hide()">
+        <!-- text will be inserted via jQuery at runtime -->
```

Comment:
The _refresh_ link is on the top of the box now. Personally, it like it more an the bottom. 

![image](https://user-images.githubusercontent.com/4305769/149787968-d0ca2bb5-a269-4288-afec-44f3b1b2ea0b.png)

Do you agree with this?
Can we move the _refresh_ link to the bottom?

Reply:
I'm almost neutral to this one ... but on a second thought you might be right 👍 
will change it.

Reply:
done, see latest commit